[API-925] Update and expand auth instructions. (#98)
cobalthq/cobalt-api-docs: versions/v2/content/authentication.md

</aside>

Our Living Documentation and API Explorer are located in [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/)

> and built with the [OpenAPI specification](https://swagger.io/specification/).  To explore our API
> from Swagger, complete the following steps:
> 
> - Create and copy your API token from your [Cobalt profile](https://app.cobalt.io/settings/api-token).
> - Go to [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/) and click the green
>   _Authorize_ button on the right side of the screen.
> - Paste your API token into the `ApiToken` input.
> - Click the green _Authorize_ button below the `ApiToken` input.  The word `Authorized` should
>   appear underneath the `ApiToken` input.
> - Close the dialog box by clicking either _Close_ button.
> - Scroll down in the right endpoints column until you see the `Orgs` section.
> - Click the `GET /orgs` endpoint to expand it.
> - Click the _Try it out_ button, then click the _Execute_ button.
> - Scroll down to the `Response body` section.  Depending on how many orgs you belong to, you should
>   see one or more `resource` sections within the output.  Each of those sections should contain a
>   `token` field.  Select the org you want to experiment with and copy the `token` field value.  This
>   is your Org Token.
> - Return back to the _Authorize_ section.
> - Paste the Org Token into the `OrgToken` input and click the green _Authorize_ button below it.
>   The word `Authorized` should appear underneath the `OrgToken` input.
> - Close the dialog box again by clicking either _Close_ button.
> - All subsequent requests to `/assets`, `/findings`, `/pentests`, etc. will be scoped to your
>   personal API token and the organization selected.

Reviewer: I'm suggesting a rewrite, based on the [Google Developer Style Guide](https://developers.google.com/style), and my personal tests of these steps. I'll be happy to explain each of my proposed changes if/as needed.

```suggestion
and built with the [OpenAPI specification](https://swagger.io/specification/).  To explore our API, complete the following steps:

- Create and copy your API token from your [Cobalt profile](https://app.cobalt.io/settings/api-token).
- Go to [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/) and select the green
  _Authorize_ button on the right side of the screen.
- Paste your API token into the `ApiToken` text box.
- Select the green _Authorize_ button below the `ApiToken` input.  You should see `Authorized` 
   below the `ApiToken` label.
- Close the Available Authorizations dialog box.
- Search for the `Orgs` endpoint.
- Select the `GET /orgs` operation to expand it.
- Select the _Try it out_ button, and then select the _Execute_ button.
- Scroll down to the `Response body`. Identify and copy the `token` associated with your target
   organization (`org`). This is your Org Token.
- Scroll up and select the _Authorize_ button.
- Paste the Org Token into the `OrgToken` text box and select the green _Authorize_ button.
  You should see `Authorized` below the `OrgToken` label.
- Close the Available Authorizations dialog box.
- The OpenAPI (Swagger) tool will now include your API Token and Org Token with your next REST calls.
```

Reviewer: Note: I'm not sure about the last step, as Swagger doesn't include the Org Token with my subsequent REST calls.

Reviewer: After discussion with @josh-hamacher-cobalt , I wonder if we should even include steps associated with the "Org Token".

... even if it's there in the `Available Authorizations` dialog box

TLDR, our implementation of Swaggerhub doesn't include the "Org Token" in subsequent REST calls.

</aside>

Our Living Documentation and API Explorer are located in [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/)
and built with the [OpenAPI specification](https://swagger.io/specification/).
To explore our API, complete the following steps:

- Create and copy your API token from your [Cobalt profile](https://app.cobalt.io/settings/api-token).
- Go to [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/) and select the green
  _Authorize_ button on the right side of the screen.
- Paste your API token into the `ApiToken` text box.
- Select the green _Authorize_ button below the `ApiToken` input.  You should see `Authorized` below
  the `ApiToken` label.
- Close the Available Authorizations dialog box.

> Most (but not all) API calls are scoped to a specific organization.  This is done via the
> `X-Org-Token` header.  Any operations that require this will have that header as a required field.

Reviewer: I like your addition. I add this suggestion to "simplify" the wording:

```suggestion
Most API calls are scoped to a specific organization, with the `X-Org-Token` header.
We include that header, as needed, in our API calls.
```

To see which organizations you belong to:

- Search for the `Orgs` endpoint.
- Select the `GET /orgs` operation to expand it.
- Select the _Try it out_ button, and then select the _Execute_ button (note that you do **not**
  need to provide the `X-Org-Token` header here).
- Scroll down to the `Response body`. Copy the `token` associated with your target organization

>   (`org`). This is your Org Token and should be provided in the `X-Org-Token` header when required.

Reviewer: ```suggestion
  (`org`). This is your Org Token. Include it with the `X-Org-Token` header when required.
```
